tutorial/source/gmm.ipynb

```diff
@@ -593,6 +594,113 @@
     "pyplot.ylabel('assignment probability')\n",
     "pyplot.legend(loc='center');"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## MCMC\n",
+    "\n",
+    "Next we'll explore the full posterior over component parameters using collapsed NUTS, i.e. we'll use NUTS and marginalize out all discrete latent variables."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Sample: 100%|██████████| 300/300 [00:30<00:00,  9.99it/s, step size=1.69e-01, acc. rate=0.587]\n"
+     ]
+    }
+   ],
+   "source": [
+    "from pyro.infer.mcmc import MCMC, NUTS\n",
+    "pyro.set_rng_seed(2)\n",
+    "kernel = NUTS(model)\n",
+    "mcmc = MCMC(kernel, num_samples=250, warmup_steps=50)\n",
+    "posterior = mcmc.run(data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "marginal = posterior.marginal(sites=[\"locs\"]).empirical[\"locs\"]\n",
```

Comment:
Just FYI - this API will be updated soon, and running mcmc (using the new interface) will simply return a dict of samples from the posterior.